### PR TITLE
made configurable url

### PR DESCRIPTION
**Pull request by markfrst**
https://github.com/atipugin/telegram-bot-ruby/issues/193

### CODE REVIEW

**Comment by markfrst** [comment]
234

**Comment by markfrst** [comment]
WIP

**Comment by markfrst** [comment]
@atipugin check it, pls  🤓

File: lib/telegram/bot/api.rb [hunk 1]
```diff
@@ -50,8 +50,9 @@ class Api # rubocop:disable ClassLength
 
       attr_reader :token
```
Comment (atipugin) [inline lib/telegram/bot/api.rb]: Please add `:url` att_reader here to make it consistent with token.

File: lib/telegram/bot/api.rb [hunk 2]
```diff
@@ -110,7 +111,7 @@ def camelize(method_name)
       end
 
       def conn
-        @conn ||= Faraday.new(url: 'https://api.telegram.org') do |faraday|
+        @conn ||= Faraday.new(url: @url) do |faraday|
```
Comment (atipugin) [inline lib/telegram/bot/api.rb]: Please replace `@url` with `url` (because we prefer attr_readers here).

**Comment by markfrst** [comment]
@atipugin fixed

**Comment by atipugin** [comment]
Thanks, merged!

**Pull request merged**